### ISSUE TITLE
Initialize Readme file

### ISSUE THREAD

**Issue by Nahrain1** [amosproj/amos2024ss07-updating-flash-boot-loader#15]
## User story
1. As a user
2. I want to get the basic informations about the project 
3. So that I get familiar with it   

## Acceptance criteria
* The team Logo is inserted
* The project description is included 
* The necessary available documentation is included in the "Getting started" section
* The Readme file is updated in Github

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
